Algorithm.CSharp/StopLossOnOrderEventRegressionAlgorithm.cs

```diff
@@ -73,16 +73,16 @@ public override void OnData(Slice data)
         public Dictionary<string, string> ExpectedStatistics => new Dictionary<string, string>
         {
             {"Total Trades", "2"},
-            {"Average Win", "0.00%"},
-            {"Average Loss", "0%"},
-            {"Compounding Annual Return", "0.273%"},
+            {"Average Win", "0%"},
+            {"Average Loss", "0.00%"},
+            {"Compounding Annual Return", "-0.156%"},
             {"Drawdown", "0.000%"},
-            {"Expectancy", "0"},
-            {"Net Profit", "0.003%"},
+            {"Expectancy", "-1"},
+            {"Net Profit", "-0.002%"},
             {"Sharpe Ratio", "0"},
             {"Probabilistic Sharpe Ratio", "0%"},
-            {"Loss Rate", "0%"},
-            {"Win Rate", "100%"},
+            {"Loss Rate", "100%"},
+            {"Win Rate", "0%"},
```

Comment:
Think we should review  _why_ it went from `"Loss Rate", "0%"` to `"Loss Rate", "100%"`?

Reply:
In master, the stop order is filled at the bid which is higher than the current price (win rate of 100%). In the PR, the stop order is filled at the open of the next bar which price is close to the market fill price.